docs/source/python/models/ssd.md

```diff
@@ -0,0 +1,8 @@
+# Ssd
```

Comment:
@sovrasov just for reference. I think we could move `ssd.py` to `detection_model.py` in the future.

Reply:
I see your point, but not 100% sure it makes sense to have both abstract `DetectionModel` and inherited from it `SSD` in one file. In opposite, in the future we might want to add other detection models. At the first iteration let's try to save compatibility of imports/includes